### PR TITLE
Added libu2f_server recipe

### PR DESCRIPTION
**Pull request by alwayslivid**
Relevant pull requests: #4497 #4493

Link: https://developers.yubico.com/libu2f-server/
GitHub repository: https://github.com/Yubico/libu2f-server

### CODE REVIEW

File: app-crypt/libu2f_server/libu2f_server-1.1.0.recipe [hunk 1]
```diff
@@ -0,0 +1,86 @@
+SUMMARY="Yubico universal 2nd factor (U2F) server C library"
+DESCRIPTION="libu2fhost is a server-side implementation \
+of the U2F protocol.
+It provides an API for generating the JSON blobs that are
+required for the U2F Registration and U2F Authentication
+operations. Moreover, it provides functionality for the
+verification of the cryptographic operations."
```
Comment (Begasus) [inline app-crypt/libu2f_server/libu2f_server-1.1.0.recipe]: Check the 80 character limit for the lines in DESCRIPTION, keep paragraphs together with a backslash at the end of a line, move new paragraphs to  new line.

File: app-crypt/libu2f_server/libu2f_server-1.1.0.recipe [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libjson_c$secondaryArchSuffix
+	lib:libhidapi$secondaryArchSuffix
+	lib:libssl$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libu2f_server_devel$secondaryArchSuffix = $portVersion
```
Comment (korli) [inline app-crypt/libu2f_server/libu2f_server-1.1.0.recipe]: should be libu2f_server${secondaryArchSuffix}_devel

File: app-crypt/libu2f_server/libu2f_server-1.1.0.recipe [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+PROVIDES="
+	libu2f_server$secondaryArchSuffix = $portVersion
+	cmd:u2f_server$commandSuffix = $portVersion
+	lib:libu2f_server$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libjson_c$secondaryArchSuffix
+	lib:libhidapi$secondaryArchSuffix
```
Comment (Begasus) [inline app-crypt/libu2f_server/libu2f_server-1.1.0.recipe]: Looking at it locally (build) it doesn't use libhidapi (here and for devel)

File: app-crypt/libu2f_server/libu2f_server-1.1.0.recipe [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libjson_c$secondaryArchSuffix
+	devel:libhidapi$secondaryArchSuffix
+	devel:libssl$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:autoconf
```
Comment (Begasus) [inline app-crypt/libu2f_server/libu2f_server-1.1.0.recipe]: Sugggestion:
```diff
-	cmd:autoconf
+	cmd:aclocal
+	cmd:autoreconf
+	cmd:awk
```

**Comment by korli** [comment]
is this a GCI task?

**Comment by Begasus** [comment]
Don't think so

**Comment by korli** [comment]
> Don't think so

Maybe we should create one :)

**Comment by alwayslivid** [comment]
> is this a GCI task?

Yes

**Comment by korli** [comment]
> > is this a GCI task?
> 
> Yes

Link please

**Pull request merged**